### PR TITLE
CFE-3569: sys.os_name_human not Unknown

### PR DESCRIPTION
**Pull request by larsewi**
Test that sys.os_name_human does not resolve to unknown _(see [CFE-3569](https://tracker.mender.io/browse/CFE-3569))_.

### CODE REVIEW

File: tests/acceptance/01_vars/01_basic/os_name_human_not_unknown.cf [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+{
+  meta:
+    "description" -> { "CFE-3569" }
+      string => "Make sure sys.os_name_human does not resolve to 'Unknown'";
+}
+
+bundle agent check
+{
+  classes:
+      "passed" expression => not(strcmp("$(sys.os_name_human)", "Unknown"));
```
Comment (olehermanse) [inline tests/acceptance/01_vars/01_basic/os_name_human_not_unknown.cf:16]: Can you also test that it is not empty string `""` and does not contain `os_name_human`?

Comment (olehermanse) [inline tests/acceptance/01_vars/01_basic/os_name_human_not_unknown.cf:16]: Can you also test that it is not empty string `""` and does not contain `os_name_human`?

**Review by olehermanse — CHANGES_REQUESTED** [review]

File: tests/acceptance/01_vars/01_basic/os_name_human_not_unknown.cf (hunk 1 above)
Comment (larsewi) [reply tests/acceptance/01_vars/01_basic/os_name_human_not_unknown.cf:16]: Sure thing !

**Comment by larsewi** [comment]
@cf-bottom, Jenkins please!

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/6276/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/6276/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-6276/

**Review by vpodzime — APPROVED** [review]
Looks good to me.

**Review by olehermanse — APPROVED** [review]
Great, thanks!
